lib/gosu/compat.rb

```diff
@@ -103,24 +103,26 @@ def self.from_text(*args)
   end
 end
 
-# No need to pass a Window to Sample.
-class Gosu::Sample
-  alias initialize_without_window initialize
-
+# No need to pass a Window to Sample and even better don't use Sample at all.
+class Gosu::Sample < Gosu::Sound
```

Comment:
Inheritance because we got the double deprecation, but maybe just replace the whole construct with deprecate_const?

Reply:
+1 deprecate_const, Gosu::Sound's interface should be compatible with Sample's